### PR TITLE
pep8 coding style conformance for scripts/unocommands.py

### PR DESCRIPTION
**Pull request by MucahidAydin**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [x] Code is properly formatted
- [x] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [x] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by mrkara** [comment]
Hello @MchdAydin47 ! :)

Thank you for your pull request! You can fix the missing change-id issue by building the code, or if that's not possible, by running the githook script: `./scripts/refresh-git-hooks` (this is needed only once for each cloned repo)

Then amending your commit: `git commit --amend`

And while already there, you may also want to adjust [the commit message](https://serdarkuzucu.com/2020/04/19/git-commit-mesaji-nasil-yazilmali/) a bit. ;)

**Comment by MchdAydin47** [comment]
@mrkara  I fix it, thank you

File: scripts/unocommands.py [hunk 1]
```diff
@@ -12,8 +12,10 @@
 import sys
 from lxml import etree
 
+
 def usageAndExit():
-    message = """usage: {program} [--check|--update|--translate] online_dir [...]
```
Comment (mrkara) [inline scripts/unocommands.py:16]: I think this line needs to stay as it is. This is part of help text, telling user about the command-line options. Please revert the change on this line.

File: scripts/unocommands.py [hunk 2]
```diff
@@ -34,10 +36,12 @@ def usageAndExit():
     {program} --translate /path/to/online /path/to/translations
 
 """
-    print(message.format(program = sys.argv[0]))
+    print(message.format(program=sys.argv[0]))
     exit(1)
 
 # Extract uno commands name from lines like "  'Command1', 'Command2',"
+
```
Comment (mrkara) [inline scripts/unocommands.py:43]: These two new blank lines separate the function description above from its function below. Please remove these new empty lines. And move the comment above into the function as in the example below:

```python
def quadratic(a, b, c, x):
    """Use the quadratic formula"""
    x_1 = (- b+(b**2-4*a*c)**(1/2)) / (2*a)
    x_2 = (- b-(b**2-4*a*c)**(1/2)) / (2*a)

    return x_1, x_2
```

File: scripts/unocommands.py [hunk 3]
```diff
@@ -48,26 +52,30 @@ def commandsFromLine(line):
...
-                commands += [ command ]
+                commands += [command]
                 command = ''
         elif inCommand:
             command += c
 
     return commands
 
 # Extract uno commands name from lines like "  {uno: '.uno:Command3',"
+
```
Comment (mrkara) [inline scripts/unocommands.py:63]: Same as above. Don't separate description from the function, please.

File: scripts/unocommands.py [hunk 4]
```diff
@@ -48,26 +52,30 @@ def commandsFromLine(line):
...
 
     m = re.search(r"\b_UNO\('.uno:([^']*)'", line)
     if m:
-        return [ m.group(1) ]
+        return [m.group(1)]
 
     return []
 
 # Extract all the uno commands we are using in the Online menu
+
```
Comment (mrkara) [inline scripts/unocommands.py:77]: Same as above. Don't separate description from the function, please.

File: scripts/unocommands.py [hunk 5]
```diff
@@ -83,6 +91,8 @@ def extractMenuCommands(path):
     return set(commands)
 
 # Extract all the uno commands we are using in the Online context menu
+
```
Comment (mrkara) [inline scripts/unocommands.py:94]: Same as above. Don't separate the description from the function, please.

File: scripts/unocommands.py [hunk 6]
```diff
@@ -113,11 +123,14 @@ def extractContextCommands(path):
...
             commands += commandFromMenuLine(line)
 
-    commands = [command for command in commands if command not in commandsToIgnore]
+    commands = [command for command in commands
+                if command not in commandsToIgnore]
     # may the list unique
     return set(commands)
 
 # Extract all the uno commands we are using in the Online toolbar
+
```
Comment (mrkara) [inline scripts/unocommands.py:132]: Same as above. Don't separate the description from the function, please.

File: scripts/unocommands.py [hunk 7]
```diff
@@ -186,9 +201,11 @@ def extractToolbarCommands(path):
     return set(commands)
 
 # Create mapping between the commands and appropriate strings
+
```
Comment (mrkara) [inline scripts/unocommands.py:204]: Same as above. Don't separate the description from the function, please.

File: scripts/unocommands.py [hunk 8]
```diff
@@ -198,18 +215,18 @@ def collectCommandsFromXCU(xcu, descriptions, commands, label, type):
...
+                                                 'office.org/2001/registry', })
             if len(textElement) == 1:
                 # extract the uno command's English text
                 text = ''.join(textElement[0].itertext())
                 command = {}
                 if unoCommand in descriptions.keys():
                     command = descriptions[unoCommand]
 
-                if not type in command:
+                if type not in command:
```
Comment (mrkara) [inline scripts/unocommands.py:229]: I'm not sure about this change.

File: scripts/unocommands.py [hunk 9]
```diff
@@ -226,12 +243,16 @@ def collectCommandsFromXCU(xcu, descriptions, commands, label, type):
     return descriptions
 
 # Print commands from all the XCU files, and collect them too
-def writeUnocommandsJS(onlineDir, lofficeDir, menuCommands, contextCommands, toolbarCommands):
+
```
Comment (mrkara) [inline scripts/unocommands.py:246]: Same as above. Don't separate the description from the function, please.

File: scripts/unocommands.py [hunk 10]
```diff
@@ -303,7 +348,7 @@ def writeUnocommandsJS(onlineDir, lofficeDir, menuCommands, contextCommands, too
 window.removeAccessKey = function(text) {
 \t// Remove access key markers from translated strings
 \t// 1. access key in parenthesis in case of non-latin scripts
-\ttext = text.replace(/\(~[A-Za-z]\)/, '');
+\ttext = text.replace(/\\(~[A-Za-z]\\)/, '');
```
Comment (mrkara) [inline scripts/unocommands.py:351]: Let's revert this for now?

File: scripts/unocommands.py [hunk 11]
```diff
@@ -313,10 +358,13 @@ def writeUnocommandsJS(onlineDir, lofficeDir, menuCommands, contextCommands, too
     return descriptions
 
 # Read the uno commands present in the unocommands.js for checking
+
```
Comment (mrkara) [inline scripts/unocommands.py:361]: Same as above. Don't separate the description from the function, please.

File: scripts/unocommands.py [hunk 12]
```diff
@@ -330,18 +378,22 @@ def parseUnocommandsJS(onlineDir):
     return strings
 
 # Generate translation JSONs for the .uno: commands
+
```
Comment (mrkara) [inline scripts/unocommands.py:381]: Same as above. Don't separate the description from the function, please.

**Review by mrkara — CHANGES_REQUESTED** [review]
Hi @MucahidAydin ! :)

Thanks for fixing the change-id issue and updating the commit message. Nice changes overall but there are still some minor points to fix. Could you please do the necessary changes, then

- amend you commit: `git commit -a --amend`
- push again: `git push --force-with-lease`

Thanks!

File: scripts/unocommands.py [hunk 13]
```diff
@@ -407,13 +452,19 @@ def writeTranslations(onlineDir, translationsDir, strings):
...
         processedCommands = set(written.keys())
 
     # check that we have translations for everything
-    dif = (menuCommands | contextCommands | toolbarCommands) - processedCommands
+    dif = (menuCommands | contextCommands or
+           toolbarCommands) - processedCommands
+
     if len(dif) > 0:
-        sys.stderr.write("ERROR: The following commands are not covered in unocommands.js, run scripts/unocommands.py --update:\n\n.uno:" + '\n.uno:'.join(dif) + "\n\n")
+        sys.stderr.write("ERROR: The following commands",
```
Comment (mrkara) [inline scripts/unocommands.py:464]: You can't slice the input like this. write() takes only 1 argument, but now you are giving 4.

Comment (MucahidAydin) [reply scripts/unocommands.py]: I made the change thank you : )

**Review by mrkara — APPROVED** [review]

**Comment by mrkara** [comment]
@MucahidAydin Thank you for your contribution, and congratulations on your first pull request merged! :)

We're looking forward to your next pull request. ;)
